app/src/main/res/values/strings.xml

```diff
@@ -80,6 +80,8 @@
     <string name="route_search">Weg suchen</string>
     <string name="partner_search">Partner suchen</string>
     <string name="rock_relevance_filter">Bedeutung: mindestens</string>
+    <string name="route_grade_filter">Schwierigkeitseinschätzung:</string>
```

Comment:
We should name it "Mittlere Schwierigkeitseinschätzung" because you calculate the average.

Reply:
In this case, we would need this for each filter

Reply:
Maybe.